frontend/src/components/core/Sidebar/ThemedSidebar.tsx

```diff
@@ -34,7 +34,6 @@ const createSidebarTheme = (theme: ThemeConfig): ThemeConfig =>
   )
 
 const ThemedSidebar = ({
-  theme,
```

Comment:
This was just unused, or?

Reply:
yeah, as far as I can tell it was simply unused